spec/JSON-RPC.md

```diff
@@ -1561,6 +1562,39 @@ No parameters
 
 [Back to **List of methods**](#list-of-methods)
 
+## chain_getPossibleAuthors
+Returns the list of the account that can generate the blocks at the given block number.
```

Comment:
Returns the list of accounts that can generate the blocks at the given block number.